content/en/docs/pipelines/sdk/python-function-components.ipynb

```diff
@@ -0,0 +1,508 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">  # For operations with a single return
&nbsp; # value, the output reference can be accessed as task.output or
&nbsp; # task.outputs['output_name'].
</pre><p> </p><p>Optional nit: do we also want to give some hints on the multiple-output case here? For example, please find details below if you're using component with multiple outputs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/website/pull/2290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>